### PR TITLE
Schedule page

### PR DESCRIPTION
**Pull request by homer0**
## What does this PR do?

Adds the schedule page to the site.

## How should it be tested manually?

Go to `/schedule` and compare with the [spec](https://xd.adobe.com/spec/5428a082-b1e3-4782-5c0c-4108e7c69e91-2219/).

### CODE REVIEW

File: src/data/constants/speakers.js [hunk 1]
```diff
@@ -17,12 +20,18 @@ Villa Allende.
 En **Una historia de layouts** nos invita a recorrer la historia desde el punto de
 vista del desarrollador a la hora de implementar una nueva estructura o layout. Nos va a
 presentar las herramientas que tenemos, las que se vienen y las que marcan el futuro.
+    `,
+    scheduleDescription: `
+Vamos a recorrer la historia desde el punto de vista del desarrollador a la hora de implementar una nueva estructura o layout. Nos va a presentar las herramientas que tenemos, las que se vienen y las que marcan el futuro.
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Vamos a recorrer la historia desde el punto de vista del desarrollador a la hora de implementar una nueva estructura o layout, descubriendo las herramientas que tenemos, las que se vienen y las que marcan el futuro.
```

File: src/data/constants/speakers.js [hunk 2]
```diff
@@ -37,12 +46,18 @@ mentorear, ayudar y los perritos.
 En **GG Easy con Cypress, testeando una app entera en media hora (no clickbait)** nos va a explicar
 cuándo y por qué hacer tests de integración o unitarios, y qué ventajas tiene Cypress sobre
 Selenium o Puppeteer.
+    `,
+    scheduleDescription: `
+Se va a explicar cuándo y por qué hacer tests de integración o unitarios, y qué ventajas tiene Cypress sobre Selenium o Puppeteer.
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Vamos a explicar cuándo y por qué hacer tests de integración o unitarios, y qué ventajas tiene Cypress sobre Selenium o Puppeteer.
```

File: src/data/constants/speakers.js [hunk 3]
```diff
@@ -76,12 +97,18 @@ Engineer en Mercado Libre.
 En **El mejor amigo para tu frontend - BFF** compartirá algunos aprendizajes obtenidos en la
 implementación del patrón BFF (Backend for Frontend). Se adentrará en los desafíos, decisiones
 importantes y tips a tener en cuenta.
+    `,
+    scheduleDescription: `
+Se compartirá algunos aprendizajes obtenidos en la implementación del patrón BFF (Backend for Frontend). Se adentrará en los desafíos, decisiones importantes y tips a tener en cuenta.
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Compartiremos algunos aprendizajes obtenidos en la implementación del patrón BFF (Backend for Frontend), adentrándonos en los desafíos, decisiones importantes y tips a tener en cuenta.
```

File: src/data/constants/speakers.js [hunk 4]
```diff
@@ -114,12 +147,18 @@ el fino arte de entender a los usuarios.
 En **Diseñando el código** nos va a contar que son los design systems, en qué consiste una style
 guide y como armar una component library, explorando como diseñadores y desarrolladores trabajan
 en conjunto.
-`
+    `,
+    scheduleDescription: `
+Veremos que son los design systems, en qué consiste una style guide y como armar una component library, explorando como diseñadores y desarrolladores trabajan en conjunto.
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Veremos qué son los design systems, en qué consiste una style guide y cómo armar una component library, explorando la manera en que diseñadores y desarrolladores trabajan en conjunto.
```

File: src/data/constants/speakers.js [hunk 5]
```diff
@@ -129,12 +168,18 @@ En **Entendiendo las PWA, componentes y funcionalidades**, nos contará en qué
 Marcos se dedica al desarrollo web desde hace aproximadamente 6 años y actualmente trabaja en Naranja, como arquitecto frontend.
 
 En **Entendiendo las PWA, componentes y funcionalidades**, nos va a contar parte por parte cómo se constituye una Progressive Web App, desde service workers y manifests hasta Workbox.
+  `,
+    scheduleDescription: `
+Se vera parte por parte cómo se constituye una Progressive Web App, desde service workers y manifests hasta Workbox.
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Veremos parte por parte cómo se constituye una Progressive Web App, desde service workers y manifests hasta Workbox.
```

File: src/data/constants/speakers.js [hunk 6]
```diff
@@ -144,12 +189,18 @@ En su charla **Irrepresentabilidad de estados inválidos** en JavaScript nos va
 Pablo es Ingeniero en Sistemas y trabaja hace más de 10 años en diversas áreas de IT (backend, frontend, UX y más). Hincha de River, guitarrista y estrella de rock frustrada.
 
 En su charla **Irrepresentabilidad de estados inválidos** en JavaScript nos va a contar acerca de las fortalezas del tipado estático y cómo su implementación contribuye a una mejora en la calidad del código.
-`
+    `,
+    scheduleDescription: `
+Cuales son las fortalezas del tipado estático y cómo su implementación contribuye a una mejora en la calidad del código
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Las fortalezas del tipado estático y cómo su implementación contribuye a una mejora en la calidad del código y, con un poco de suerte, a una reducción en la cantidad de errores en producción.
```

File: src/data/constants/speakers.js [hunk 7]
```diff
@@ -159,6 +210,9 @@ En su charla **GraphQL 101** nos va a contar de que trata GraphQL y como se comp
 Alexis es Ingeniero en Sistemas de Información. Empezó su carrera como desarrollador Front-end y desde entonces trabajó con tecnologías tanto Front como Back. En la actualidad, se desempeña como líder de equipo en Virtualmind.
 
 En su charla **GraphQL 101** nos va a contar de que trata GraphQL y como se compara con otras formas de definir una API en el backend.
-`
+    `,
+    scheduleDescription: `
+De que trata GraphQL y como se compara con otras formas de definir una API en el backend
```
Comment (joelalejandro) [inline src/data/constants/speakers.js]:
```suggestion
Veremos de qué se trata GraphQL y cómo se compara con otras formas de definir una API en el backend.
```

**Comment by asermax** [comment]
Btw, what happened with the sponsors section? Are we adding the rest of the sponsors on this PR or a different one?

**Comment by homer0** [comment]
@asermax

1. The bug: I didn't notice it. I'll check it tonight.
2. Sponsors: The spec wasn't updated when I finished this; @joelalejandro told me he was going to update it today, so I'm assuming I'll be handling that on a different PR.

**Comment by joelalejandro** [comment]
I'm merging this so we can release it as is. It can be fixed on develop.

**Pull request merged**